x-pack/plugin/ml-feature-index-builder/src/main/java/org/elasticsearch/xpack/ml/featureindexbuilder/action/TransportDeleteFeatureIndexBuilderJobAction.java

```diff
@@ -31,6 +33,7 @@
         extends TransportMasterNodeAction<DeleteFeatureIndexBuilderJobAction.Request, AcknowledgedResponse> {
 
     private final PersistentTasksService persistentTasksService;
+    private static final Logger logger = LogManager.getLogger(TransportDeleteFeatureIndexBuilderJobAction.class.getName());
```

Comment:
I think it's more usual to just pass the class rather than the name, so `LogManager.getLogger(TransportDeleteFeatureIndexBuilderJobAction.class)` in this case.

Reply:
I second this, other than this, the PR looks good. Will approve when changed.